pytest.ini

```diff
@@ -14,4 +14,5 @@ minversion = 6.0
 log_format = %(asctime)s.%(msecs)03d %(levelname)s [%(filename)s:%(lineno)d] %(message)s
 log_date_format = %Y-%m-%d %H:%M:%S
 log_cli = true
+; log_cli_level = debug ; uncomment to see more verbose output
```

Comment:
There is a CLI arg for this. I'm not sure this addition is necessary, but up to you